docs/flow/reference/components/prometheus.scrape.md

```diff
@@ -6,26 +6,70 @@ title: prometheus.scrape
 
 # prometheus.scrape
 
-`prometheus.scrape` configures a metrics scraping job for a given set of
+`prometheus.scrape` configures a Prometheus scraping job for a given set of
 `targets`. The scraped metrics are forwarded to the list of receivers passed in
 `forward_to`.
 
 Multiple `prometheus.scrape` components can be specified by giving them
 different labels.
 
+## Scraping behavior
+The `prometheus.scrape` component borrows the scraping behavior of Prometheus.
+Prometheus, and by extend this component, uses a pull model for scraping
+metrics from a given set of _targets_.
+Each scrape target is defined as a set of _labels_ i.e. key-value pairs. The
+set of targets can either be _static_, or dynamically provided periodically by
+a service disovery component such as `discovery.kubernetes`. The special
+internal label `__address__` _must always_ be present and corresponds to the
+`<host>:<port>` that will be used for the scrape request.
+
+By default, the scrape job will try to scrape all available targets' `/metrics`
```

Comment:
```suggestion
By default, the scrape job tries to scrape all available targets' `/metrics`
```

Reply:
Google ref: https://developers.google.com/style/tense?hl=en

Reply:
I'm doing this _a lot_, aren't I? 😅  thanks for the heads up!

Reply:
Haha, it's tricky because these things are happening in the future -- when the software is run. But the present tense is definitely more concise.